homeassistant/components/overkiz/quality_scale.yaml

```diff
@@ -0,0 +1,68 @@
+rules:
```

Comment:
```py
OverkizSensorDescription(
        key=OverkizState.CORE_TEMPERATURE,
        name="Temperature",
        native_value=lambda value: round(cast(float, value), 2),
        device_class=SensorDeviceClass.TEMPERATURE,
        # core:MeasuredValueType = core:TemperatureInCelcius
        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
        state_class=SensorStateClass.MEASUREMENT,
    ),
```
Why do we round? If its a noisy entity that is fine! Otherwise I would suggest removing the `round` and just setting a `suggested_precision`

Reply:
What do you mean with noisy? There are sometimes entities that have more than 2 decimals, where we use the round function here to make sure we only update it in this range. I am not too familiar with the suggested precision, what would be the benefit here?